packages/discovery-provider/integration_tests/tasks/test_index_payment_router.py

```diff
@@ -96,7 +98,12 @@
                 "7dw7W4Yv7F1uWb9dVH1CFPm39mePyypuCji2zxcFA556": 1000000,
             },
             "total_price_cents": 200,
-            "access": PurchaseAccessType.stream,
+        },
+        {  # download access type
+            "track_id": 3,
+            "splits": {"7gfRGGdp89N9g3mCsZjaGmDDRdcTnZh9u3vYyBab2tRy": 1000000},
```

Comment:
gah I should have followed the constants pattern from test_user_bank_indexer to make this stuff easier to follow.